python/ctranslate2/extensions.py

```diff
@@ -267,7 +269,9 @@ def translator_generate_tokens(
 
 def generator_generate_tokens(
     generator: Generator,
-    prompt: List[str],
+    start_tokens: Union[List[str], List[List[str]]],
+    max_batch_size: int = 32,
```

Comment:
I think a better default behavior is to pass the input batch as-is, without rebatching.

```suggestion
    max_batch_size: int = 0,
```